File: src/com/jepl/lang/Runner.java
package com.jepl.lang;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;

public class Runner {
    public static void runFile(String file){
        String text;
        try {
            text = Files.readString(Path.of(file));
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
        Interpreter.interpret(text);
        if(Values.funs.containsKey("main")){
            Interpreter.interpret(Values.getFun("main"));
        }
    }

    public static void runString(String input){
        Interpreter.interpret(input);
        if(Values.funs.containsKey("main")){
            Interpreter.interpret(Values.getFun("main"));
        }
    }

    public static double langVersion = 1.1;
}

File: src/com/jepl/lang/Runner.kt
package com.jepl.lang

import java.io.File
import java.io.IOException

class Runner {
    companion object {
        val langVersion: String
            get() = "1.34"

        fun runFile(file: String) {
            val codeOf: String
            try {
                codeOf = File(file).readText()
            }catch (e: IOException) {
                throw JEPLException(e)
            }
            Interpreter.interpret(codeOf)

            if (Values.funs.containsKey("main")) {
                Interpreter.interpret(Values.getFun("main").toString())
            }
        }

        fun runString(code: String) {
            Interpreter.interpret(code)
            if (Values.funs.containsKey("main")) {
                Interpreter.interpret(Values.getFun("main").toString())
            }
        }
    }
}